app/assets/javascripts/admin/user/user_list_view.js

```diff
@@ -246,6 +247,17 @@ class UserListView extends React.PureComponent<Props, State> {
               dataIndex="email"
               key="email"
               sorter={Utils.localeCompareBy("email")}
+              render={(__, user: APIUserType) => (
+                <span>
+                  <EditableTextLabel
+                    value={user.email}
+                    onChange={newEmail => {
+                      user.email = newEmail;
+                      updateUser(user);
+                    }}
```

Comment:
@hotzenklotz after changing the email, the frontend still shows the old email because it does not reload the page. I tried 'this.fetchData()' but it didn't help. How do you tell React to reload the data?